src/server/mock-data/json/klachten.json

```diff
@@ -0,0 +1,60 @@
+{

```

Comment:
In deze file graag de omschrijving anonimiseren / verkorten zodat het niet meer herleidbaar naar een persoon. Daarnaast, iemand kan letterlijk de inhoud van zijn klacht herkennen